Spell/__init__.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+
+"""
+Check spelling of any word in any language you want. Example: spell en great
+"""
+
+from albertv0 import Item
+from albertv0 import ClipAction
+import os
+import re
+import subprocess
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "Spell checker"
+__version__ = "2.0"
+__trigger__ = "spell "
+__author__ = "Marek Mazur"
+__dependencies__ = []
+
+
+dict_file = '~/.local/share/albert/org.albert.extension.python/modules/Spell/dictionaries/{language}.dict'
```

Comment:
you should better use another way to get the ~/.local/share directory. see https://askubuntu.com/questions/457047/how-can-i-get-the-xdg-default-user-directories-from-python

Reply:
use [this](https://albertlauncher.github.io/docs/extensions/python/#extension-interface-02) to get the path prefixes

Reply:
fixed